.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: ci
+
+# This worflow needs this secret:
+#
+# DOCKERPASSWORD = Docker Hub token
+
+on:
+  push:
+    branches: [master]
+    tags: 'v*.*.*'
```

Comment:
This means, this workflow runs on pushes to ```master``` and if you make a tag with that pattern. But not if you push to ```devel``` you would have to change line 9 to `branches: [master, devel]` to do so.

Reply:
Aye, it's clear now. :+1: